### PR TITLE
done 

### PR DESCRIPTION
**Pull request by ali1373107**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

-----
[View rendered README.md](https://github.com/ali1373107/JavaScript-Core-1-Coursework-Week1/blob/master/README.md)
[View rendered exercises/B-hello-world/README.md](https://github.com/ali1373107/JavaScript-Core-1-Coursework-Week1/blob/master/exercises/B-hello-world/README.md)

### CODE REVIEW

**Comment by ali1373107** [comment]
Hi @Hamisakim i done this home work you can have review on it

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Like learning a musical instrument, programming requires daily practise.
 
-The exercises are split into three folders: `exercises`, `mandatory` and `extra`. All homework in the `exercise` and `mandatory` section **must** be completed for homework by the following lesson. 
+The exercises are split into three folders: `exercises`, `mandatory` and `extra`. All homework in the `exercise` and `mandatory` section **must** be completed for homework by the following lesson.0
```
Comment (Hamisakim) [inline README.md:5]: 0 ?

File: exercises/B-hello-world/README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Inside of `exercise.js` there's a line of code that will print "Hello world!".
 
 - Open a terminal window
 - Change directory to this folder (`cd exercises/B-hello-world`) - assuming you're at the project root
-- Run the program using node (`node exercise.js`)
+- Run the program using node (`c)
```
Comment (Hamisakim) [inline exercises/B-hello-world/README.md:11]: did you mean to do this

File: exercises/I-floats/exercise.js [hunk 3]
```diff
@@ -1,2 +1,7 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var roughStudent = Math.round(numberOfStudents)
+var roughMentor = Math.round(numberOfMentors)
+console.log(roughStudent)
+console.log(roughMentor)
+//I could not solve this problem
```
Comment (Hamisakim) [inline exercises/I-floats/exercise.js:7]: Math.round() rounds a decimal to the nearest whole number. you need to convert to percentages 

```
const total = numberOfMentors + numberOfStudents;
const percentageOfStudents = (numberOfStudents / total) * 100;
const percentageOfMentors = (numberOfMentors / total) * 100;
const roundedPercentageOfStudents = Math.round(percentageOfStudents);
const roundedPercentageOfMentors = Math.round(percentageOfMentors);
const message1 = "Percentage students: " + roundedPercentageOfStudents + "%";
const message2 = "Percentage mentors: " + roundedPercentageOfMentors + "%";
console.log(message1);
console.log(message2);
```

File: exercises/J-functions/exercise.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
 function halve(number) {
-  // complete the function here
+  return number*2;
```
Comment (Hamisakim) [inline exercises/J-functions/exercise.js:2]: this doubles, not halves

File: exercises/K-functions-parameters/exercise.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 // Complete the function so that it takes input parameters
-function multiply() {
-  // Calculate the result of the function and return it
+function multiply(num1, num2) {
+  return num1* num2
 }
 
 // Assign the result of calling the function the variable `result`
```
Comment (Hamisakim) [inline exercises/K-functions-parameters/exercise.js:6]: you haven't done this part!

Comment (Hamisakim) [reply exercises/K-functions-parameters/exercise.js:6]: call the function as a variable

File: exercises/K-functions-parameters/exercise4.js [hunk 6]
```diff
@@ -1,5 +1,7 @@
 // Declare your function first
-
+function add(num1,num2){
+    return num1+num2;
+}
 // Call the function and assign to a variable `sum`
-
+sum = add(13,123);
```
Comment (Hamisakim) [inline exercises/K-functions-parameters/exercise4.js:6]: add const

File: exercises/K-functions-parameters/exercise3.js [hunk 7]
```diff
@@ -1,4 +1,7 @@
 // Write your function here
+function createGreeting(name){
+    return name
```
Comment (Hamisakim) [inline exercises/K-functions-parameters/exercise3.js:3]: this just returns the name not a string

File: mandatory/1-syntax-errors.js [hunk 8]
```diff
@@ -1,16 +1,18 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
-  return a + b + c;
+function addNumbers(a, b, c) {
+  sum = a + b + c
```
Comment (Hamisakim) [inline mandatory/1-syntax-errors.js:4]: const sum = a + b + c

File: mandatory/1-syntax-errors.js [hunk 9]
```diff
@@ -1,16 +1,18 @@
...
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age){
+  return "Hello, my name is " + name + " and I am " + age + " years old";
+}
 
 function getTotal(a, b) {
-  total = a ++ b;
+  total = a + b;
```
Comment (Hamisakim) [inline mandatory/1-syntax-errors.js:13]: const total = a + b

File: mandatory/4-tax.js [hunk 10]
```diff
@@ -5,7 +5,12 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  sale = price +((price /100) *20) 
+  return sale;
+}
+s = calculateSalesTax(100)
```
Comment (Hamisakim) [inline mandatory/4-tax.js:12]: add const

File: mandatory/4-tax.js [hunk 11]
```diff
@@ -17,7 +22,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+    prudactPrice= price +((price /100) *20)
```
Comment (Hamisakim) [inline mandatory/4-tax.js:26]: const productPrice = price +((price /100) *20)
watch out for spelling.  Install the spell check extension i showed you :)

File: mandatory/4-tax.js [hunk 12]
```diff
@@ -17,7 +22,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+    prudactPrice= price +((price /100) *20) 
+    p= "£" ;
```
Comment (Hamisakim) [inline mandatory/4-tax.js:27]: what is ths for?

File: mandatory/4-tax.js [hunk 13]
```diff
@@ -17,7 +22,13 @@ function calculateSalesTax() {}
...
+    return p + parseFloat(prudactPrice).toFixed(2);
+}
+p = calculateSalesTax(120.890)
```
Comment (Hamisakim) [inline mandatory/4-tax.js:30]: const p = calculateSalesTax(120.890)

**Review by Hamisakim — COMMENTED** [review]
Make sure you read the question and take your time. You missed a couple of questions!!

* Not your fault because the instructions used it, but going forward always use `let` or `const` instead of `var` 
*  install the spell check extension 

Nice work Ali ! 👍

**Pull request closed without merging**